src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useRef, useState } from 'react';
+import { Errors } from '../../enums/Errors';
+import { useTodosContext } from '../../helpers/useTodoContext';
+import classNames from 'classnames';
+import { USER_ID, addTodos } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+import { handleErrors } from '../../helpers/hendleErorrs';
+
+export const TodoForm = () => {
+  const {
+    todos,
+    setTodos,
+    completedTodos,
+    activeTodos,
+    setErrorMessage,
+    setTempTodo,
+    setLoadingTodoIds,
+    loadingTodoIds,
+    shouldFocus,
+    setShouldFocus,
+  } = useTodosContext();
+  const addTodoInputRef = useRef<HTMLInputElement>(null);
+  const [title, setTitle] = useState('');
+
+  const isClassActive = completedTodos.length > 0 && activeTodos.length === 0;
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setErrorMessage(null);
+    setTitle(event.target.value);
+  };
+
+  const addTodo = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!title.trim().length) {
+      handleErrors(Errors.EmptyTitle, setErrorMessage);
```

Comment:
Let's remove `else` block by using this code
```suggestion
    if (!title.trim().length) {
      handleErrors(Errors.EmptyTitle, setErrorMessage);
      return;
```